lua/telescope/_extensions/file_browser/finders.lua

```diff
@@ -62,9 +62,13 @@ local function fd_file_args(opts)
   if not opts.respect_gitignore then
     table.insert(args, "--no-ignore-vcs")
   end
+  if opts.no_ignore then
+    table.insert(args, "--no-ignore")
+  end
   if opts.follow_symlinks then
     table.insert(args, "--follow")
   end
+  vim.print(args)
```

Comment:
Debug print left in accidentally? Or is this intended?

Reply:
that was unintended :grin: 
thanks for taking a look